framer/SVGGroup.coffee

```diff
@@ -66,23 +66,8 @@ class SVGGroup extends Layer
 
 	@defineGroupProxyProp "fill"
 	@defineGroupProxyProp "stroke"
-	@defineGroupProxyProp "strokeWidth", _.isNumber, parseInt, Object.prototype.toString
-	@defineGroupProxyProp "strokeWidthMultiplier", _.isNumber, parseInt, Object.prototype.toString
+	@defineGroupProxyProp "strokeWidth", _.isNumber, parseInt
+	@defineGroupProxyProp "strokeWidthMultiplier", _.isNumber, parseInt
 	@defineGroupProxyProp "color", Color.validColorValue, Color.toColor
 
-	@define "gradient",
```

Comment:
No gradient on groups? (Is it part of the fill/stroke?)

Reply:
Currently we don't support gradients on shapes at all, we will implement this in the future.